lib/tether/ops_linux.go

```diff
@@ -893,10 +894,18 @@ func (t *BaseOperations) MountTarget(ctx context.Context, source url.URL, target
 		return fmt.Errorf("unable to create mount point %s: %s", target, err)
 	}
 
-	rawSource := source.Hostname() + ":/" + source.Path
+	var rawSource bytes.Buffer
```

Comment:
purely for readability I would do the following:

src := filepath.Join(source.Hostname() + ":", source.Path)

filepath.Join controls number of leading slashes,
https://play.golang.org/p/cx4yphPIN3h

Reply:
It's also significantly slower and with extra allocations; neither of which will be significant in this case.
I mostly did not want to rely on `Clean` (which Join calls) because I don't want to impose actual path content restrictions on people.